js/app/controllers/editorcontroller.js

```diff
@@ -124,15 +124,15 @@ app.controller('EditorController', ['$scope', 'TimezoneService', 'AutoCompletion
 		$scope.validate = function() {
 			var error = false;
 			if ($scope.properties.summary === null || $scope.properties.summary.value.trim() === '') {
-				OC.Notification.showTemporary(t('calendar', 'Please add a title!'));
+				OC.Notification.showTemporary(t('calendar', 'Please add a title.'));
 				error = true;
 			}
 			if ($scope.calendar === null || typeof $scope.calendar === 'undefined') {
-				OC.Notification.showTemporary(t('calendar', 'Please select a calendar!'));
+				OC.Notification.showTemporary(t('calendar', 'Please pick a calendar.'));
 				error = true;
 			}
 			if (!$scope.properties.checkDtStartBeforeDtEnd()) {
-				OC.Notification.showTemporary(t('calendar', 'The event ends before it starts!'));
+				OC.Notification.showTemporary(t('calendar', 'The event can not end before it starts.'));
```

Comment:
I think it is a, while polite, way that makes it harder for people whom are not English to understand it.
Entertain the idea that "may not" could mean it is not 100% clear cut that the event ends before it starts. As in maybe. And yes, into context you could derive that this is not the intention, but it makes it possible to get into that sort of thinking.